### PR TITLE
#193 Duplicate part in the pester test is removed.

### PR DESCRIPTION
**Pull request by ArianArt**

### CODE REVIEW

**Review by OlegTokar — APPROVED** [review]

File: Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/BackgroundTask/AddIshBackgroundTask.cs [hunk 1]
```diff
@@ -67,7 +67,7 @@ namespace Trisoft.ISHRemote.Cmdlets.BackgroundTask
     /// <code>
     /// New-IshSession -WsBaseUrl "https://example.com/ISHWS/"
     /// $ishBackgroundTask = Get-IshDocumentObj -LogicalId "GUID-ABCD-1234" |
-    ///                      Add-IshBackgroundTask -EventType "SYNCHRONIZEMETRICS" -InputDataTemplate IshObjectWithIshRef
+    ///                      Add-IshBackgroundTask -EventType "SYNCHRONIZEMETRICS" -InputDataTemplate IshObjectWithLngRef
```
Comment (ddemeyer) [inline Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/BackgroundTask/AddIshBackgroundTask.cs:70]: For example `Get-IshDocumentObj -LogicalId ISHEDITORTEMPLATEMODULECONCEPT` will return multiple versions and multiple languages.  So the description is wrong mentioned "latest". I did not check outside of this PR, but the typical case for SYNCHRONIZEMETRICS should be -InputDataTemplate IshObjectsWithIshRef where the server expands the LogicalId, right?

File: Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/Settings/GetIshTypeFieldDefinition.Tests.ps1 [hunk 2]
```diff
@@ -71,26 +71,13 @@ Describe "Get-IshTypeFieldDefinition" -Tags "Read" {
...
-		    	(Get-IshTypeFieldDefinition -IshSession $ishSession | Where-Object -Property ISHType -EQ 'ISHEvent' | Where-Object -Property AllowOnCreate -EQ $false).Count | Should -Be 11 # all columns are explicit api parameters and cannot be set over metadata
-		    	(Get-IshTypeFieldDefinition -IshSession $ishSession | Where-Object -Property ISHType -EQ 'ISHEvent' | Where-Object -Property IsMultiValue -EQ $false).Count | Should -Be 23 # all columns are single value
-		    	(Get-IshTypeFieldDefinition -IshSession $ishSession | Where-Object -Property ISHType -EQ 'ISHEvent' | Where-Object -Property IsSystem -EQ $true).Count | Should -Be 23 # all columns are system columns
-		    	(Get-IshTypeFieldDefinition -IshSession $ishSession | Where-Object -Property ISHType -EQ 'ISHEvent' | Where-Object -Property Name -EQ 'USERID').DataSource | Should -Be 'ISHUser'
-            }
+		    (Get-IshTypeFieldDefinition -IshSession $ishSession | Where-Object -Property ISHType -EQ 'ISHEvent' | Where-Object -Property Level -EQ 'Progress').Count | Should -Be 11
```
Comment (ddemeyer) [inline Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/Settings/GetIshTypeFieldDefinition.Tests.ps1:74]: It is okay, but I don't like updating tests that actually don't belong to story #193 ... you better submit them separately under #4

**Review by ddemeyer — CHANGES_REQUESTED** [review]

File: Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/Settings/GetIshTypeFieldDefinition.Tests.ps1 (hunk 2 above)
Comment (ArianArt) [reply Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/Settings/GetIshTypeFieldDefinition.Tests.ps1:74]: Actually, this one is more related to the issue #191

Comment (ddemeyer) [reply Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/Settings/GetIshTypeFieldDefinition.Tests.ps1:74]: Indeed so better to submit is as PR for #191 as that would increase traceability a lot. It is okay to leave for now, but clustering per story helps in the long run...

**Review by ddemeyer — APPROVED** [review]

**Pull request merged**